1-ECommerce/src/components/Filters.tsx

```diff
@@ -0,0 +1,41 @@
+import { Menu } from './Menu'
+import { Searchbar } from './Searchbar';
+import { SortBy } from './SortBy';
+
+export const Filters = () => {
+
+    const sortOptions = [
+        { id: 1, label: 'Sory By' },
+        { id: 2, label: 'Favorites' },
+        { id: 3, label: 'A-Z' },
+        { id: 4, label: 'Z-A' },
+        { id: 5, label: 'Low Prices' },
+        { id: 6, label: 'High Prices' },
+    ]
+
+    const options = sortOptions
+        .map((option) => {
+            return (
+                <option
+                    key={option.id}
+                    value={option.id}
+                >{option.label}</option>
+            )
+        })
```

Comment:
this can be defined outside of the component, since it doesn't need any of its props or state. We could define a new component for it or just move it to line 4.

Reply:
Also, we can remove the return and the `{ }` inside the map